### PR TITLE
jknjnkjjknjk

### LINKED ISSUE #4: Hasanboy
hbhjbhbhjbhjhjbhjbhjbhjbjhbhjjhb

### PR DESCRIPTION
**Pull request by HasanboyXomidov**
Closes #4

### CODE REVIEW

File: ConsoleApp1/ConsoleApp1/HasanboyClass.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    public class HasanboyClass
+    {
+        public static string GetHasanboy()
+        {
+            return "Hasanboy";
```
Comment (HasanboyXomidov) [inline ConsoleApp1/ConsoleApp1/HasanboyClass.cs:13]: Yaxshi yozilgan

Comment (Maxsetovich) [reply ConsoleApp1/ConsoleApp1/HasanboyClass.cs:13]: good job bro

Comment (Maxsetovich) [inline ConsoleApp1/ConsoleApp1/HasanboyClass.cs:13]: dvfbtgbt.

Comment (HasanboyXomidov) [reply ConsoleApp1/ConsoleApp1/HasanboyClass.cs:13]: thanks

**Review by Maxsetovich — CHANGES_REQUESTED** [review]

**Pull request merged**